### ISSUE TITLE
No documentation

### ISSUE THREAD

**Issue by ylecuyer** [Antergos/lightdm-webkit2-greeter#16]
Why there is no documentation about the lightdm api provided in JavaScript?

**Comment by sbalneav** [comment]
I think you can probably close this now, for two reasons:

1) The manpage now has most of the API documented.
2) Now that we've gotten the API in sync with the LightDM api, that can also act as a reference.

**Comment by lots0logs** [comment]
Yeah, though I already [started on it](https://antergos.com/wiki/development/lightdm-webkit2-greeter-theme-javascript-api/) so I might as well finish it :wink:
